src/locale/lang/id.js

```diff
@@ -0,0 +1,86 @@
+export default {
+  el: {
+    datepicker: {
+      now: 'Sekarang',
+      today: 'Hari ini',
```

Comment:
Thanks for contributing!

There may not be enough space for the eight-lettered`Hari ini`, do you have any shorthand for it in Indonesian?